### PR TITLE
Fix leading dims in tuning scripts and other minor fixes

### PR DESCRIPTION
**Pull request by amdkila**
- Wait until after branching to merge

### CODE REVIEW

File: tuning/automation/GenerateTuningConfigurations.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def processFile(headerFileName, key, configDefinitionList, configurationPath, wo
 
 def SetDefaultStrides(problemDefinition, m, n, k):
     if problemDefinition["transposeB"] == "T":
-        return [k, k, n]
+        return [m, n, m]
```
Comment (zaliu) [inline tuning/automation/GenerateTuningConfigurations.py:98]: If transposeA is true here, default lda would be k, not m, [k, n, m] should be returned.  Instead it returns [m, n, m] in error.

Comment (amdkila) [reply tuning/automation/GenerateTuningConfigurations.py:98]: are you referring to TT sizes? I've made some assumptions that we don't encounter TT sizes

Comment (zaliu) [reply tuning/automation/GenerateTuningConfigurations.py:98]: > are you referring to TT sizes? I've made some assumptions that we don't encounter TT sizes

Then please add a brief comment to SetDefaultStrides stating the assumption.  Thanks.

Comment (amdkila) [reply tuning/automation/GenerateTuningConfigurations.py:100]: Good idea. I added that in my next commit

**Pull request merged**